src/content/docs/misc/sending-messages.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Sending Messages
+---
+
+The basic feature of the Discord Bots is to send messages all around Discord. And in Seyfert you can send them in the easiest way.
+
+First of all, we have to setup a basic `Hello World` command.
+
+```ts title="src/commands/helloworld.ts" showLineNumbers
+import {
+	Command,
+	Declare
+} from "seyfert";
+
+@Declare({
+	name: "helloworld",
+	description: "Send a basic hello world message."
+})
+
+
+export default class HelloWorldCommand extends Command {
+	async run(ctx: CommandContext) {}
+}
+```
+
+Having set up our basic `Hello World` command we are ready to send our first message using `CommandContext.write()` function.
+
+```ts title="src/commands/helloworld.ts" showLineNumbers
+export default class HelloWorldCommand extends Command {
+	async run(ctx: CommandContext) {
+        ctx.write({ content: "Hello World 👋" })
+    }
+}
+```
+
+The `CommandContext.write()` function will respond to the command.
+
+## EditOrReply
+
+But, what about responding to the command or editing it's reply instead of of sending only messages where the command was triggered?
+
+We can use `CommandContext.editOrReply()` function. This function is used to reply the command or, if the response has been sent, edit it. 
+
+This function is very useful if we want to develop a command which responds to the command or if the command was responded the response will be edited. If we are only using a bare `CommandContext.write()` we will send a reply in all the cases.
+
+Here is an example of how implement this function. 
+
+```ts title="src/commands/helloworld.ts" showLineNumbers
+export default class HelloWorldCommand extends Command {
+	async run(ctx: CommandContext) {
+        ctx.editOrReply({ content: "Hello World 👋" }) //if we respond to the command before executing this function the response will be edited instead of sent.
+    }
+}
+```
+
+## Sending messages without a reply
+
+Reading this guide you could have thought about the possibility of only send a message to a channel instead of replying to the command.
+
+Here we are. To send a simple message to a specific channel we need to retrive it's id and then access to `BaseClient.messages` property and run the `write` function.
+
+Here is an example of how to send that message without replying a command:
+
+```ts title="src/commands/helloworld.ts" showLineNumbers
+export default class HelloWorldCommand extends Command {
+	async run(ctx: CommandContext) {
+        ctx.client.messages.write("channelId", { content: "Hello World 👋" })
```

Comment:
here instead of use "channelId" as placeholder you could get the actual channel the interaction is begin created